rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -10045,15 +10047,36 @@ void opcode_0362(const ScriptArguments& args, const ScriptCharacter character, S
 	@arg coord Coordinates
 	@arg radius Radius
 	@arg model Model ID
-	@arg arg6 Boolean true/false
+	@arg visible Boolean true/false
 */
-void opcode_0363(const ScriptArguments& args, ScriptVec3 coord, const ScriptFloat radius, const ScriptModel model, const ScriptBoolean arg6) {
-	RW_UNIMPLEMENTED_OPCODE(0x0363);
-	RW_UNUSED(coord);
-	RW_UNUSED(radius);
-	RW_UNUSED(model);
-	RW_UNUSED(arg6);
-	RW_UNUSED(args);
+void opcode_0363(const ScriptArguments& args, ScriptVec3 coord, const ScriptFloat radius, const ScriptModel model, const ScriptBoolean visible) {
+	auto& models = args.getVM()->getFile()->getModels();
+	auto& modelName = models[-model];
```

Comment:
I agree.. however, instead of fixing it here we should just cross fingers imo and create a prioritized issue about proper `ScriptModel` / `args.getModel()` instead.

This also happens elsewhere in the code and another piece of code uses `modes[abs(index)]` (which felt even worse)